### ISSUE TITLE
Cmus doesn't build with FFMPEG4 because LIBAVCODEC_VERSION_MAJOR is used where LIBAVUTIL_VERSION_MAJOR should be used instead

### ISSUE THREAD

**Issue by MIvanchev** [cmus/cmus#1341]
See [this patch](https://github.com/void-linux/void-packages/pull/51317/files#diff-7c96145f4b86e93877d915f1d7a6beea97731a6901c8669acc439d754c319a31) for the Void Linux package. 

The function `av_opt_set_chlayout` is related to libavutil, not libavcodec, so the correct constant has to be checked in `ip/ffmpeg.c`.

**Comment by gavtroy** [comment]
It's the same issue as #1329. AVCodec and AVUtil are so tightly coupled that I'm not sure why there are separate version defines or that it should _really_ matter which one we use, although obviously I did mess it up. :roll_eyes: I wrote something similarly confused in my comment on the other issue.

AVChannelLayout is defined in libavutil/channel_layout.h so your description and fix are good, but since we're accessing it through an [AVCodecContext](https://git.ffmpeg.org/gitweb/ffmpeg.git/blob/cdba98bb80:/libavcodec/avcodec.h#l2051) (also part of the API update) I'd say `LIBAVCODEC_VERSION_MAJOR` is still technically correct. I went that direction in #1330.
